go/merge/three_way_test.go

```diff
@@ -59,3 +71,20 @@ func valToTypesValue(f func(seq) types.Value, v interface{}) types.Value {
 	}
 	return v1
 }
+
+func TestThreeWayMerge_PrimitiveConflict(t *testing.T) {
```

Comment:
Why did you not include this in ThreeWayMergeSuite? Just didn't need the stuff from Setup/TearDown or some other reason?  Otherwise LGTM.